$NetBSD: patch-ad,v 1.3 2005/09/27 11:25:37 dsainty Exp $

Standard NetBSD curses is acceptable.  We also need libi386 for the direct I/O
access.

--- Makefile	2002-05-21 03:20:02.000000000 +1200
+++ Makefile	2005-09-13 02:41:17.000000000 +1200
@@ -10,9 +10,8 @@
   CC:=${CC}
 endif
 
-CFLAGS	= -I/usr/include/ncurses -O2 -Wall -pipe -g
-LDFLAGS = -lmenu -lncurses
-LIBS	= -lncurses
+CFLAGS	= -O2 -Wall -pipe -g
+LIBS	= -lcurses -li386
 
 PICOBJS	= picprg.o lowlvl.o pichex.o
 
